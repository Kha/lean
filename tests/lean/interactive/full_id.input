{"command": "sync", "file_name": "f", "content": "eval tt"}
{"command": "check"}
{"command": "info", "line": 1}
