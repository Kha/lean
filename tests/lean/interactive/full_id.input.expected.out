{"message":"new file name, reloading","response":"ok"}
{"is_ok":true,"messages":[{"caption":"eval result","file_name":"f","pos_col":0,"pos_line":1,"severity":"information","text":"tt"}],"response":"ok"}
{"messages":[{"column":5,"full_id":"bool.tt","kind":"full_id","line":1}],"response":"ok"}
